dune

```diff
@@ -0,0 +1,2 @@
+(dirs
+ (:standard \ _esy node_modules example))
```

Comment:
I'm not very practiced at all with native, what does adding this Dune file do for us?

Reply:
https://dune.readthedocs.io/en/stable/dune-files.html#dirs-since-1-6